Drivers/ManagePersonsPermissionCheckerTaskDisplayDriver.cs

```diff
@@ -0,0 +1,21 @@
+using Lombiq.TrainingDemo.Activities;
+using Lombiq.TrainingDemo.ViewModels;
+using OrchardCore.Workflows.Display;
+using OrchardCore.Workflows.Models;
+
+namespace Lombiq.TrainingDemo.Drivers;
+
+// ActivityDisplayDriver is specifically for implementing Workflow Activity Tasks.
+// Don't forget to register this class with the service provider (see: Startup.cs).
+public class ManagePersonsPermissionCheckerTaskDisplayDriver :
```

Comment:
Here are code formatting issues that the analyzers will show you once you merge in OSOCE and the build runs.